### ISSUE TITLE
Using an unknown loglevel for logging kills the entire system

### ISSUE THREAD

**Issue by rogeralsing** [akkadotnet/akka.net#1057]
If you supply a config like `loglevel = Debug` instead of `loglevel = DEBUG` the entire system dies.
There is a warning in the log on what levels are OK, so that is good, but killing the entire system seems somewhat too strong IMO.

if you use a console logger, you will have a hard time to see why everything crashed.
Should we default unknown levels to DEBUG?

I mean, an incorrect remote address does not kill the system, so why should loglevels do that?

**Comment by Aaronontheweb** [comment]
Yeah, this is an oversight. I would log a bright red "ERROR" message for having an unknown log level (so the end-user knows that they've goofed,) but then treat all of the logging calls as debug calls thereafter.

**Comment by rogeralsing** [comment]
I'm looking at this right now.
The problem is that, when the exception occurs, there are no loggers, as they have not yet been set up.

Should we catch the exception, store it in a var, and then once loggers are set up, check if we have a stored exception and publish it?
It feels dirty, but we cant publish exceptions to loggers that are not yet existing :)

Or should we just close this issue and claim that we are in line with JVM?

**Comment by Danthar** [comment]
I would much rather have the system to be well-behaved in this regard then have it die silently. So i would vote for storing the exception in a var and publish it later when the system has started.

**Comment by rogeralsing** [comment]
I'll make a fix for this one

**Comment by numo16** [comment]
I'll take a look at this one.

**Comment by Danthar** [comment]
I confirmed that in Akka 1.0.4 the system no longer crashes on unknown loglevel.
At least when used in combination with the serilog integration. Not sure if thats relevant or not though..

However regarding the unknown loglevel, im not seeing any warning or anything.
So whats the status on this one ?
